### ISSUE TITLE
soft delete (on attributes)

### ISSUE THREAD

**Issue by mikermcneil** [balderdashy/waterline#134]
no idea how this should be set up-- probably would need to look at best practices, but just tracking it

**Comment by sailsbot** [comment]
Thanks for posting, @mikermcneil.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by rachaelshaw** [comment]
@Thomasdezeeuw @snyx1 @mphasize @polastre @aclave1 you can now soft-delete records with `.archive()` - [Docs](https://next.sailsjs.com/documentation/reference/waterline-orm/models/archive)

**Comment by abalad** [comment]
@rachaelshaw The idea of SoftDelete is cool, but we need to create more implementations.
As a BLUEPRINT to be able to perform this operation directly via ENDPOINT
